CryptoCoarseFundamentalUniverseSelectionAlgorithm.cs

```diff
@@ -35,23 +37,32 @@ public override void Initialize()
             // Data ADDED via universe selection is added with Daily resolution.
             UniverseSettings.Resolution = Resolution.Daily;
 
-            SetStartDate(2022, 2, 14);
-            SetEndDate(2022, 2, 18);
+            SetStartDate(2017, 2, 14);
+            SetEndDate(2017, 2, 18);
             SetCash(100000);
 
-            // add a custom universe data source (defaults to usa-equity)
-            AddUniverse<MyCustomDataUniverseType>("MyCustomDataUniverseType", Resolution.Daily, data =>
+            SetBrokerageModel(BrokerageName.Binance, AccountType.Cash);
+
+            // Add universe selection of cryptos based on coarse fundamentals
+            AddUniverse(new CryptoCoarseFundamentalUniverse(Market.Binance, UniverseSettings, UniverseSelectionFilter));
+        }
+
+        private IEnumerable<Symbol> UniverseSelectionFilter(IEnumerable<CryptoCoarseFundamental> data)
+        {
+            var universe = new List<Symbol>();
+
+            foreach (var datum in data)
             {
-                foreach (var datum in data)
+                Log($"{datum.Symbol},{datum.Price},{datum.Volume},{datum.DollarVolume},{datum.VolumeInBaseCurrency},{datum.Open},{datum.High},{datum.Low},{datum.Close}");
```

Comment:
Believe ToString() is already implemented, so can clean up this log? Same for the other algos 👍